platform/lib/pipeline/markdownDocument.js

```diff
@@ -212,10 +220,31 @@ class MarkdownDocument {
    * @return {Promise}
    */
   save(path) {
+    let content = '';
     const frontmatter = `---\n${yaml.safeDump(this._frontmatter, {'skipInvalid': true})}---\n\n`;
-
     path = path ? path : this._path;
-    return writeFile.promise(path, frontmatter + this._contents).then(() => {
+    content += frontmatter;
+
+    /**
+    * check if file is imported and if so add a comment in order to inform that
+    * the file should not be changed in the amp.dev/docs - repro
+    */
+
```

Comment:
Since the comment above belongs directly to the if condition there should be no space between.